### PR TITLE
support farming

### PR DESCRIPTION
**Pull request by thoka**
choose couchdb database from url

### CODE REVIEW

**Comment by WardCunningham** [comment]
@lenada, @paul90 -- Help reviewing this please.

**Comment by WardCunningham** [comment]
Related to https://github.com/fedwiki/wiki-node/pull/38

File: lib/couchdb.js [hunk 1]
```diff
@@ -10,8 +10,11 @@ module.exports = function (opts) {
         host           = config.host || '127.0.0.1',
         port           = config.port || 5984,
         classicPageGet = fsPage(opts).get,
-        connection     = new (cradle.Connection)(host, port, {cache: false}),
-        db             = connection.database(dbName);
+        connection     = new (cradle.Connection)(host, port, {cache: false});
+
+    if (opts.farm) { dbName = dbName+"-"+opts.url.split("//")[1].split(":")[0].replace(".","$"); }
```
Comment (lenada) [inline lib/couchdb.js]: @thoka @WardCunningham jshint fails: https://travis-ci.org/fedwiki/wiki-storage-couchdb/jobs/51702825#L200 :neckbeard: 
`lib/couchdb.js: line 15, col 41, Strings must use singlequote...`

**Comment by paul90** [comment]
Fixes #3

**Pull request merged**